### ISSUE TITLE
Debian gnutls-utils became gnutls-bin

### ISSUE THREAD

**Issue by stappersg** [GothenburgBitFactory/taskserver-setup#9]
Hello,

My first attempt on generating _certificates_ failed with `ERROR: No certtool found`.

Installing `gnutls-utils` is not possible, the package became `gnutls-bin`.

Screenshot,  note the `certtool` in package description of `gnutls-bin`.
```text
stappers@hc4:/usr/src/taskserver/pki
$ sudo apt install gnutls-utils
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package gnutls-utils
stappers@hc4:/usr/src/taskserver/pki
$ apt show gnutls-bin                                                              
Package: gnutls-bin
Version: 3.7.1-5
Priority: optional
Section: net
Source: gnutls28
Maintainer: Debian GnuTLS Maintainers <pkg-gnutls-maint@lists.alioth.debian.org>
Installed-Size: 1,747 kB
Depends: libc6 (>= 2.25), libgnutls-dane0 (>= 3.7.0), libgnutls30 (>= 3.7.0-0+private+1), libopts25 (>= 1:5.18.16), libtasn1-6 (>= 4.14)
Homepage: https://www.gnutls.org/
Tag: implemented-in::c, interface::commandline, network::client,
 network::server, protocol::ssl, role::program, security::cryptography,
 suite::gnu
Download-Size: 631 kB
APT-Sources: http://deb.debian.org/debian bullseye/main arm64 Packages
Description: GNU TLS library - commandline utilities
 GnuTLS is a portable library which implements the Transport Layer
 Security (TLS 1.0, 1.1, 1.2, 1.3) and Datagram
 Transport Layer Security (DTLS 1.0, 1.2) protocols.
 .
 GnuTLS features support for:
  - certificate path validation, as well as DANE and trust on first use.
  - the Online Certificate Status Protocol (OCSP).
  - public key methods, including RSA and Elliptic curves, as well as password
    and key authentication methods such as SRP and PSK protocols.
  - all the strong encryption algorithms, including AES and Camellia.
  - CPU-assisted cryptography with VIA padlock and AES-NI instruction sets.
  - HSMs and cryptographic tokens, via PKCS #11.
 .
 This package contains a commandline interface to the GNU TLS library, which
 can be used to set up secure connections from e.g. shell scripts, debugging
 connection issues or managing certificates.
 .
 Useful utilities include:
  - TLS termination: gnutls-cli, gnutls-serv
  - key and certificate management: certtool, ocsptool, p11tool
  - credential management: srptool, psktool

stappers@hc4:/usr/src/taskserver/pki
$ 
```

I'll provide a patch.

**Comment by stappersg** [comment]
> I'll provide a patch.

That patch became #10

**Comment by lauft** [comment]
Merged #10
